packages/node/__tests__/integration.spec.ts

```diff
@@ -696,6 +700,109 @@ describe('Incremental Delivery', () => {
   })
 })
 
+function md5File(path: string) {
+  return new Promise((resolve, reject) => {
+    const output = crypto.createHash('md5')
```

Comment:
sha!

Reply:
Since security is not a concern, I used the fastest algorithm here in order to prevent tests from getting too slow.